server/index.js

```diff
@@ -23,6 +23,20 @@ io.on('connection', socket => {
   sessionId = socket.id;
 });
 
+app.get('/api/chatRooms', (req, res, next) => {
+  const sql = `
+    select "name",
+           "chatId" as "id"
+      from "chatRooms";
+  `;
+  db.query(sql)
+    .then(result => {
+      const chatRooms = result.rows;
+      res.status(200).json(chatRooms);
+    })
+    .catch(err => console.error(err));
```

Comment:
You should pass the error to `next()` (and use an error handler middleware) so that the server still responds instead of leaving the client hanging.